### DISCUSSION TITLE
RAN metrics

### DISCUSSION THREAD

**Discussion by josua265** [srsran/srsRAN_Project#868] [General Help]
Hello everyone,
I would like to make these metrics available on the xapp oran sc ric.
![WhatsApp Image 2024-10-15 at 18 10 09](https://github.com/user-attachments/assets/6d66d605-a43b-42bc-81e2-26732b0434ad)
Do you have any idea how to do? Thanks

**Comment by pgawlowicz** [comment]
check here: https://github.com/srsran/oran-sc-ric

and here: https://docs.srsran.com/projects/project/en/latest/tutorials/source/near-rt-ric/source/index.html#xapps
